### PR TITLE
Logging h fixes

### PR DESCRIPTION
**Pull request by mnunberg**

### CODE REVIEW

File: .circleci/ci_package.sh [hunk 1]
```diff
@@ -15,38 +15,40 @@ fi
 # Install my fork of RAMP- need to merge it first
 pip install git+https://github.com/mnunberg/RAMP@print_version --upgrade
```
Comment (MeirShpilraien) [inline .circleci/ci_package.sh]: I know you did not changed it here but what is this version and why its not on master ?

Comment (mnunberg) [reply .circleci/ci_package.sh]: I don't have push access to that repo. The change I'm talking about with different shards though, I did locally. Also, the more relevant change is https://github.com/RedisLabsModules/RediSearch/pull/471. This PR just adds some logging fixes in the API

**Comment by MeirShpilraien** [comment]
@mnunberg can you explain what you have changed here? I see commits that I have done on this branch ...

**Comment by MeirShpilraien** [comment]
Also fix conflicts

**Comment by mnunberg** [comment]
@MeirShpilraien Look at this PR first: https://github.com/RedisLabsModules/RediSearch/pull/471 - I don't know why this branch is showing so many commits

**Pull request closed without merging**